context/app/routes_api.py

```diff
@@ -1,23 +1,45 @@
 from io import StringIO
 from csv import DictWriter
 
-from flask import Response, abort, request, render_template
+from flask import Response, abort, request, render_template, jsonify
 
 from .utils import make_blueprint, get_client, get_default_flask_data
 
 
 blueprint = make_blueprint(__name__)
 
 
-@blueprint.route('/metadata/v0/<entity_type>.tsv')
+def _get_dict_subset(d, keys_to_remove):
+    return {k: d[k] for k in d.keys() - dict.fromkeys(keys_to_remove)}
```

Comment:
`d.keys() - dict.fromkeys(keys_to_remove)` seems like it's a `set` - `dict`? I may be confused.

```suggestion
    return {k: v for k, v in d.items() if k not in keys_to_remove}
```
or 
```suggestion
    return {k: d[k] for k in d.keys() - keys_to_remove}
```